Bottom: 9225856745e61bcd24dfd092549bb1966b8d7f49
Top:    cf6b8797fed4116cdc21d416632ab46fee817f2e
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-03 10:44:42 +0800

Refresh of vhost-bypass-wrap_counter

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 7a4c6e6..a27af60 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -1426,10 +1426,12 @@ long vhost_vring_ioctl(struct vhost_dev *d, unsigned int ioctl, void __user *arg
 		vq->last_avail_idx = s.num;
 		/* Forget the cached index value. */
 		vq->avail_idx = vq->last_avail_idx;
+#if 0
 		if (vhost_has_feature(vq, VIRTIO_F_RING_PACKED)) {
 			vq->last_avail_wrap_counter = wrap_counter;
 			vq->avail_wrap_counter = vq->last_avail_wrap_counter;
 		}
+#endif
 		break;
 	case VHOST_GET_VRING_BASE:
 		s.index = idx;
@@ -1459,8 +1461,10 @@ long vhost_vring_ioctl(struct vhost_dev *d, unsigned int ioctl, void __user *arg
 			break;
 		}
 		vq->last_used_idx = s.num;
+#if 0
 		if (vhost_has_feature(vq, VIRTIO_F_RING_PACKED))
 			vq->last_used_wrap_counter = wrap_counter;
+#endif
 		break;
 	case VHOST_GET_VRING_USED_BASE:
 		s.index = idx;
